README.md

```diff
@@ -6,19 +6,19 @@ Github repository: <https://github.com/movestore/DistanceMoved>
 
 ## Description
 
-Calculation of the cumulative distance, net displacement or the maximum net displacement per chosen time interval or for the entire tracking period.
+Calculate the cumulative distance moved, net displacement, or maximum net displacement across the data at a chosen time interval (for example, daily net displacements) or for the entire period of each track.
 
 ## Documentation
 
-This app calculates the distance moved per chosen time interval or for the entire tracking period. The distance can be calculated in three ways:
+This app calculates the distance moved by each track per a chosen time interval or for the entire tracking period. The distance can be calculated one of in three ways:
 
-***Cumulative distance***: it calculates the sum of the length of all segments per chosen time interval or the entire track.
+***Cumulative distance*** calculates the sum of the length of all segments (straight-line distance between consecutive location records in a track) per chosen time interval or the entire track.
 
-***Net displacement***: it calculates the distance of the straight line between the 1st and the last point per chosen time interval or the entire track.
+***Net displacement*** calculates the distance of the straight line between the first and the last point per chosen time interval or the entire track.
 
-***Maximum net displacement***: it returns the maximum straight line distance of the distance matrix between all pairs of locations per chosen time interval or the entire track.
+***Maximum net displacement*** calculates the maximum straight-line distance of the distance matrix between all pairs of locations per chosen time interval or the entire track.
 
-In the 'logs' a message will be displayed informing about the time zone of the data.
+The intervals will be defined by rounding timestamps to the chosen interval and determining track segments (consecutive location records) from the data set that fall within each interval. The break point between intervals is defined at the start of the interval: "00" for seconds, minutes or hours; midnight ("00:00:00") UTC on the day; and midnight on the first day of the month or year. The 'logs' will include a message noting the time zone of the data.
```

Comment:
```suggestion
The intervals will be defined by rounding timestamps to the chosen interval and determining track segments (consecutive location records) from the data set that fall within each interval. The break point between intervals is defined at the start of the interval: "00" for seconds, minutes or hours; midnight ("00:00:00") UTC on the day; and midnight on the first day of the month or year. 

The 'logs' will include a message noting the time zone of the data, this can be important for the interpretation of the results. For example if the selected time unit is hour, to correctly interpret what time of the day the animal is experiencing it is important to know in which timezone these hours are represented.
```